pacientes/models.py

```diff
@@ -194,35 +195,34 @@ def create_from_sss(cls, dni):
                     tipo_documento=data['Tipo de documento'],
                     numero_documento=data['Número de documento'],
                 )
-                if tam_tabla >= 2:
-                    oss_data = tablas[1]['data']
-                    oss_codigo = (''.join(filter(str.isdigit, oss_data['Código de Obra Social'])))
-                    oss, created = ObraSocial.objects.get_or_create(
-                        codigo=oss_codigo,
-                        defaults={
-                            'nombre': oss_data['Denominación Obra Social'],
+                oss_data = tablas[1]['data']
+                oss_codigo = (''.join(filter(str.isdigit, oss_data['Código de Obra Social'])))
+                oss, created = ObraSocial.objects.get_or_create(
+                    codigo=oss_codigo,
+                    defaults={
+                        'nombre': oss_data['Denominación Obra Social'],
+                        })
+                ObraSocialPaciente.objects.create(
+                    data_source=settings.SOURCE_OSS_SSS,
+                    paciente=paciente,
+                    obra_social_updated=now(),
+                    obra_social=oss,
+                    tipo_beneficiario=data['Parentesco'].lower()
+                )
```

Comment:
```suggestion
                ossp, created = ObraSocialPaciente.objects.get_or_create(
                    data_source=settings.SOURCE_OSS_SSS,
                    paciente=paciente,
                    obra_social=oss
                    )
                ossp.tipo_beneficiario=data['Parentesco'].lower()
                ossp.obra_social_updated=now()
```
Creo que sería mejor no agregarlo cada vez y mantener un registro por cada OSS+ paciente y solo actualizar lo que cambia

Reply:
entiendo sisi, yo en realidad hice dos funciones esa es la de creación, se supone que si creas un paciente creas la obra social paciente..  la otra funcion si existe el paciente directamente revisa la fecha de la ultima actualización del paciente y toma los datos de la obra social haciendo eso